WTWSMS/localisation/WtWSMS_titles.csv

```diff
@@ -1110,72 +1110,74 @@ e_egypt;Egyptian Empire;Empire 
-b_asoristan;Asoristan;;;;;;;;;;x;
-b_betdarai;Betdarai;;;;;;;;;;x;
-b_vehardashir;Veh Ardashir;;;;;;;;;;x;
-b_wandanigan;Wandanigan;;;;;;;;;;x;
-b_dastkart;Dastkart;;;;;;;;;;x;
-b_hambar;Hambar;;;;;;;;;;x;
-b_vuzurgshapur;Vuzurgshapur;;;;;;;;;;x;
-b_syarzur;Syarzur;;;;;;;;;;x;
-b_shahrkart;Shahrkart;;;;;;;;;;x;
-b_arbil;Arbil;;;;;;;;;;x;
-b_arbayestan;Arbayestan;;;;;;;;;;x;
-b_betzabdai;Beth Zabdai;;;;;;;;;;x;
-b_saokoros;Saokoros;;;;;;;;;;x;
-b_sigriane;Sigriane;;;;;;;;;;x;
-b_badiyatashsham;Badiyat Ash Sham;;;;;;;;;;x;
-b_rima;Rima;;;;;;;;;;x;
-b_hira;Hira;;;;;;;;;;x;
+####### WtWSMS_eastern_expansion.csv;;;;;;;;;;;x
+k_kashgar;Kashgar;;;;;;;;;;x
+k_kashgar_adj;Kashgar;;;;;;;;;;x
+k_khotan_adj;Khotanese;Khotanais;;;;;;;;;x
+k_shanshan;Shanshan;;;;;;;;;;x
+k_shanshan_adj;Shanshani;;;;;;;;;;x
+k_kucha;Kucha;;;;;;;;;;x
+k_kucha_adj;Kuchan;;;;;;;;;;x
+k_karasahr;Karasahr;;;;;;;;;;x
+k_karasahr_adj;Karasahrli;;;;;;;;;;x
+k_turfan;Turfan;;;;;;;;;x
+k_turfan_adj;Turfani;;;;;;;;;;x
+k_jimasa;Jimasa;;;;;;;;;;x
+k_tuyuhun;Tuyuhun;;;;;;;;;;x
+k_tuyuhun_adj;Tuyuhun;;;;;;;;;;x
+d_tiele;Tiele;;;;;;;;;;x
+d_tiele_adj;Tiele;;;;;;;;;;x
+k_northern_liang;Northern Laing;Liang du Nord;;;;;;;;;x
+k_northern_liang_adj;Liang;;;;;;;;;;x
+k_gaochang;Gaochang;;;;;;;;;;x
+d_thracian_goths;Thracian Goths;;;;;;;;;;x
+d_thracian_goths_adj;Thraco-Gothic;;;;;;;;;;x
+###### WtWsMS persian expansion ##########
+c_ohrmazdardashir;Ohrmazd Ardashir;;;;;;;;;;x
+c_vahman;Vahman;;;;;;;;;;x
+c_belabad;Belabad;;;;;;;;;;x
+c_nihavand;Nihavand;;;;;;;;;;x
+c_orukit;Orukit;;;;;;;;;;x
+c_kaskar;Kaskar;;;;;;;;;;x
+c_asoristan;Asoristan;;;;;;;;;;x
+c_betdarai;Betdarai;;;;;;;;;;x
+c_vehardashir;Veh Ardashir;;;;;;;;;;x
+c_wandanigan;Wandanigan;;;;;;;;;;x
+c_dastkart;Dastkart;;;;;;;;;;x
+c_hambar;Hambar;;;;;;;;;;x
+c_vuzurgshapur;Vuzurgshapur;;;;;;;;;;x
+c_syarzur;Syarzur;;;;;;;;;;x
+c_shahrkart;Shahrkart;;;;;;;;;;x
+c_arbil;Arbil;;;;;;;;;;x
+c_arbayestan;Arbayestan;;;;;;;;;;x
+c_bethzabdai;Beth Zabdai;;;;;;;;;;x
+c_saokoros;Saokoros;;;;;;;;;;x
+c_sigriane;Sigriane;;;;;;;;;;x
+c_badiyatashsham;Badiyat Ash Sham;;;;;;;;;;x
+c_rima;Rima;;;;;;;;;;x
+c_hira;Hira;;;;;;;;;;x
+b_ohrmazdardashir;Ohrmazd Ardashir;;;;;;;;;;x
+b_vahman;Vahman;;;;;;;;;;x
+b_belabad;Belabad;;;;;;;;;;x
+b_nihavand;Nihavand;;;;;;;;;;x
+b_orukit;Orukit;;;;;;;;;;x
+b_kaskar;Kaskar;;;;;;;;;;x
+b_asoristan;Asoristan;;;;;;;;;;x
+b_betdarai;Betdarai;;;;;;;;;;x
+b_vehardashir;Veh Ardashir;;;;;;;;;;x
+b_wandanigan;Wandanigan;;;;;;;;;;x
+b_dastkart;Dastkart;;;;;;;;;;x
+b_hambar;Hambar;;;;;;;;;;x
+b_vuzurgshapur;Vuzurgshapur;;;;;;;;;;x
+b_syarzur;Syarzur;;;;;;;;;;x
+b_shahrkart;Shahrkart;;;;;;;;;;x
+b_arbil;Arbil;;;;;;;;;;x
+b_arbayestan;Arbayestan;;;;;;;;;;x
+b_betzabdai;Beth Zabdai;;;;;;;;;;x
+b_saokoros;Saokoros;;;;;;;;;;x
+b_sigriane;Sigriane;;;;;;;;;;x
+b_badiyatashsham;Badiyat Ash Sham;;;;;;;;;;x
+b_rima;Rima;;;;;;;;;;x
+b_hira;Hira;;;;;;;;;;x
+DRUHTINAZ;Druhtinaz;;;;;;;;;;x
+DRUHTINAZ_FOA;Great Warlord;;;;;;;;;;x
```

Comment:
@loup99 

Did you see the notes? "Druht**ī**naz" is a High German/Protogerman/Gothic title meaning "suffix for a _leader of a war band_". It's a flavor title!